app/Template/dashboard/subtasks.php

```diff
@@ -10,6 +10,7 @@
             <th class="column-20"><?= $paginator->order(t('Project'), 'project_name') ?></th>
             <th><?= $paginator->order(t('Task'), 'task_name') ?></th>
             <th><?= $paginator->order(t('Subtask'), \Kanboard\Model\SubtaskModel::TABLE.'.title') ?></th>
+            <?= $this->hook->render('template:dashboard:subtasks:header', array('paginator' => $paginator)) ?>
```

Comment:
The name of this hook should be more specific because this is included between specific columns.